### ISSUE TITLE
where is the CRF code ?

### ISSUE THREAD

**Issue by youyi-jia** [YudeWang/UNet_segmentation#4]
Do you really use CRF? Why didn't I find the part of CRF in the code?Is it still softmax after the last fc layer that you used directly?Because I see tF.nn.softmax in the code.Can you help me?
`   net_fc = conv_layer(net_res_conv8, 512, class_num, k_size=1, name='fc', relu=False, bn=False, stride=1)

    net = factory.UNet_ResNet(x, class_num)

    net_sub = tf.slice(net, [0, 0, 0, 1], [1, 256, 256, 4])

    net_softmax = tf.nn.softmax(net_sub) `

**Comment by YudeWang** [comment]
https://github.com/YudeWang/UNet-Satellite-Image-Segmentation/blob/09e2e279c2fa84c9d809390849632f3115cbaf35/test.py#L80
